src/components/Comments.vue

```diff
@@ -0,0 +1,96 @@
+<template>
+  <div class="mb-5" v-if="get(comments, 'length', 0)">
```

Comment:
you can just add comments.length > 0

Reply:
in unele cazuri apare eroare, de asta am pus cu get
![image](https://user-images.githubusercontent.com/28743656/132001703-f629027b-17ef-4ebd-8eaa-ef1af226dc05.png)